### PR TITLE
[TGBOT-178] Create Fine class

### PR DESCRIPTION
**Pull request by lamaconda**
https://i18nteam.atlassian.net/browse/TGBOT-178

### CODE REVIEW

**Review by MarinaSauchuk — APPROVED** [review]
lgfm

File: src/main/java/com/example/demo/model/Fine.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    public String toString() {
+        return "Fine{" +
+                "fineBalance=" + fineBalance +
+                ", reason='" + reason + '\'' +
+                ", daysLate=" + daysLate +
+                ", blackListed=" + blackListed +
+                '}';
+    }
+
+    public double calculateFine(double fineBalance) {
```
Comment (travka677) [inline src/main/java/com/example/demo/model/Fine.java:76]: Do we need this method?

**Review by travka677 — APPROVED** [review]

File: src/main/java/com/example/demo/model/Fine.java (hunk 1 above)
Comment (max-java) [reply src/main/java/com/example/demo/model/Fine.java:76]: according to SOLID principles - not )
but it is ok to left it here for now, cose our purpose now just to merge some code into git

**Review by max-java — APPROVED** [review]
lgfm

**Review by QewBye48 — APPROVED** [review]

**Review by Grideron — APPROVED** [review]

**Pull request merged**